nucleus/modelci/data_transfer_objects/eval_function.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional
+
+from pydantic import validator
+
+from ...pydantic_base import ImmutableModel
+from ..constants import ThresholdComparison
+
+
+class EvaluationCriterion(ImmutableModel):
+    """
+    eval_function_id: ID of evaluation function
+    threshold_comparison: comparator for evaluation. i.e. threshold=0.5 and threshold_comparator > implies that a test only passes if score > 0.5.
+    threshold: numerical threshold that together with threshold comparison, defines success criteria for test evaluation.
```

Comment:
Does this show up as we want in the autodocs?

Reply:
Example usage would be helpful in the docstring.

Reply:
It does not! https://scale-ai-nucleus-python-sdk--176.com.readthedocs.build/en/176/api/nucleus/modelci/index.html#nucleus.modelci.EvaluationCriterion

We'd need to indent these under a `Parameters:` header and as a best practice, add a oneliner summary of the class. Also because pydantic isn't yet supported by our autodoc framework, we'd need to add types manually.
```suggestion
    <oneline summary>
    
    Parameters:
        eval_function_id (str): ID of evaluation function
        threshold_comparison (:class:`ThresholdComparison`): comparator for evaluation. i.e. threshold=0.5 and threshold_comparator > implies that a test only passes if score > 0.5.
        threshold (float): numerical threshold that together with threshold comparison, defines success criteria for test evaluation.
```